src/components/Materialize/collapsible.js

```diff
@@ -0,0 +1,11 @@
+export default {
+  id: 'collapsible',
+  bind: () => {
+    setTimeout(() => {
+      /* eslint no-undef: 0 */
```

Comment:
This eslint pragma disables the linter for all subsequent lines of the file. From the position of this comment, I believe what you meant was: `// eslint-disable-next-line no-undef`